source/common/http/http1/conn_pool.h

```diff
@@ -51,8 +58,6 @@ class ActiveClient : public Envoy::Http::ActiveClient {
     void onAboveWriteBufferHighWatermark() override {}
     void onBelowWriteBufferLowWatermark() override {}
 
-    void onStreamDestroy();
```

Comment:
Is this just not used? Seems like nothing was calling it?

Reply:
yep.  my guess is at some point some PR started inheriting from CodecClientCallbacks, backed it out, and failed to clean it up.
I discovered when I did a pass trying to inherit from CodecClientCallbacks, realized onStreamDestroy was called before stream destruction, and backed out both my and the old functions.

Also I've got to do a change updating that name because it has bitten me twice that onStreamDestroy isn't called on stream destroy :-(